public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -107,39 +98,40 @@ include ../_util-fns
   We put a template expression within the interpolation braces when we wrote `{{1 + 1}}`.
   We’ll see template expressions again in the [property binding](#property-binding) section,
   appearing in quotes to the right of the `=` symbol as in `[property]="expression"`.
-// #enddocregion template-expressions-1
 
-// #docregion template-expressions-2
 :marked
-  We write template expressions in a language that looks like JavaScript.
-  Many JavaScript expressions are legal template expressions, but not all.
-  JavaScript expressions that have or promote side effects are prohibited,
+  We write template expressions in a language that looks like #{_JavaScript}.
+  Many #{_JavaScript} expressions are legal template expressions, but not all.
+  #{_JavaScript} expressions that have or promote side effects are prohibited,
```

Comment:
Add a blank line above this, so people are less likely to miss the context for the list.

(It originally had this blank line, but sometime before your changes the blank line disappeared.)